pkg/kube/client.go

```diff
@@ -65,10 +70,43 @@ type Client struct {
 	Log func(string, ...interface{})
 }
 
+type clientConfig struct {
+	inner clientcmd.ClientConfig
+}
+
+var _ clientcmd.ClientConfig = &clientConfig{}
+
+func (c *clientConfig) RawConfig() (clientcmdapi.Config, error) {
+	return c.inner.RawConfig()
+}
+
+func (c *clientConfig) ClientConfig() (*rest.Config, error) {
+	fmt.Printf("FUCK clientConfig.ClientConfig()\n")
```

Comment:
Fix, please. 😄  (Same with other one a bit lower in code)

Reply:
Sorry, forget to delete the debug message :stuck_out_tongue_closed_eyes: